lib/presentation/redux/states/ui_state.dart

```diff
@@ -224,7 +224,7 @@ enum ActionOutsideAppType {
 }
 
 enum ActionInsideAppType {
-  NONE, AUTHENTICATING_BIOMETRIC
+  NONE, AUTHENTICATING_BIOMETRIC,USING_CAMERA
```

Comment:
Not a good place to add action `camera` here. How about create new one?

Reply:
done